src/main/java/hudson/remoting/jnlp/Main.java

```diff
@@ -104,6 +104,25 @@ public class Main {
             usage="If the connection ends, don't retry and just exit.")
     public boolean noReconnect = false;
 
+    @Option(name = "-delay",
+            usage = "The fixed delay to occur between retries. Default is 10 seconds.")
+    public int delay = 10;
+
+    @Option(name = "-jitterFactor",
```

Comment:
(if the first comment is accepted about prefixing with retry)
```suggestion
    @Option(name = "-retryJitterFactor",
```